extension/modules/personas.js

```diff
@@ -1123,8 +1123,8 @@ var PersonaController = {
         // Update the Custom menu. Custom personas unavailable in per-window
         // personas mode.
         let customMenu = document.getElementById("personas-plus-custom-menu");
-        customMenu.hidden = this._prefs.get("perwindow") || !this._prefs.get("showCustomMenu");
-        if (!customMenu.hidden) {
+        customMenu.disabled = this._prefs.get("perwindow")
```

Comment:
Sorry yes works for me, too. Probably I did not write the steps correctly.

What about when you open New Window:

1.     Create Custom Persona with name
2.     Toggle perwindow pref to true
3.     Check toolbar button menu. Label is there.
4.     Open New Window
5.     Check toolbar button menu. Label is empty.